example/settings.py

```diff
@@ -82,9 +82,15 @@
 
 ES_DISABLED = False
 
-ES_URLS = ['http://localhost:9200']
+ES_URLS = ['http://elasticsearch.local:9200']
```

Comment:
would probably prefer to keep this general and let the host's port forwarding of 9200 handle this